pages/api/index.js

```diff
@@ -0,0 +1,17 @@
+import reverseString from '../../lib/reverseString';
+
+export default function handler(req, res) {
+    const {method, body} = req;
+    switch(method){
+        case "POST":
+            if(typeof body.data === "string"){
+                res.status(200).json({text: reverseString(body.data)});
```

Comment:
👍 I like that your request and response are keyed objects instead of just raw strings